lib/rex/compat.rb

```diff
@@ -138,7 +138,9 @@ def self.open_browser(url='http://google.com/')
   else
     # Search through the PATH variable (if it exists) and chose a browser
     # We are making an assumption about the nature of "PATH" so tread lightly
-    if defined? ENV['PATH']
```

Comment:
Not sure why `defined?` was used here as it always returns a truthy string:

```
3.0.2 :002 > defined?(ENV['missing_env_var'])
 => "method" 
```

Maybe there was a time in ruby where ENV didn't exist at all? But it's used throughout the rest of the file, so it seems out of place

Reply:
For verification in case anyone wants another example here is a run over IRB:

```
irb(main):003:1* if defined?(ENV['faleasdfasdfasdasdf'])
irb(main):004:1*   p "A"
irb(main):005:1* else
irb(main):006:1*   p "B"
irb(main):007:0> end
"A"
=> "A"                                             
irb(main):008:0>
```